src/styles/components/header.scss

```diff
@@ -0,0 +1,55 @@
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__homelink {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
```

Comment:
If nav is a separate block we need to move these styles(all related to nav) to separate file